### PR TITLE
Update readme to include community content

### PR DESCRIPTION
**Pull request by bettyjunod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +14,54 @@ debugging tools.
...
 
-## Next Steps
-- Join us on our Slack channel: [https://slack.solo.io/](https://slack.solo.io/)
-- Follow us on Twitter: [https://twitter.com/soloio_inc](https://twitter.com/soloio_inc)
-- Check out the [docs](https://docs.solo.io/service-mesh-hub/latest)
-- Check out the code and contribute: [Contribution Guide](CONTRIBUTING.md)
-- Contribute to the [Docs](docs/)
+## Community
+
+Welcome to the Service Mesh Hub Community! This is a project for people and companies interested in the service mesh. The community is open to all and please feel free to share with colleagues and friends interested in learnint more and getting involved.
```
Comment (harveyxia) [inline README.md:43]:
```suggestion
Welcome to the Service Mesh Hub Community! This is a project for people and companies interested in Service Mesh Hub. The community is open to all so please feel free to share with colleagues and friends interested in learning more and getting involved.
```

File: README.md [hunk 2]
```diff
@@ -14,31 +14,54 @@ debugging tools.
...
+
+### Contribution 
+There are many ways to get involved in an open source community and contribute to the project. Watch [this talk](https://www.youtube.com/watch?v=VE-igex6Lz4) to learn more about the architecture and how it works. 
+- **Code:** If you're looking to hack on service mesh, check out the code and the contribution guide [here](CONTRIBUTING.md) and look for the *good first issue* and *help wanted* labels in the GitHub issues. 
+ - **Docs:** Contribute to the [Docs](docs/) or file issues for any docs bugs or requests [here](https://github.com/solo-io/service-mesh-hub/issues). 
+ - **Talks and Blogs:** If you are interested in writing or speaking about Service Mesh Hub and would like access to content, images or help, [DM us here](https://solo-io.slack.com/archives/DHQ9J939V). Share your demos, tutorials and content back to the community.
+
+### Community Meetings 
+Calls will be held every other Wednesday at 10am Pacific Time and are open to the general public. TThese meetings will be recorded and posted publicly to YouTube.
```
Comment (harveyxia) [inline README.md:57]:
```suggestion
Calls will be held every other Wednesday at **10am Pacific Time** and are open to the general public. These meetings will be recorded and posted publicly to YouTube. 
```

File: README.md [hunk 3]
```diff
@@ -14,31 +14,54 @@ debugging tools.
...
+- **Code:** If you're looking to hack on service mesh, check out the code and the contribution guide [here](CONTRIBUTING.md) and look for the *good first issue* and *help wanted* labels in the GitHub issues. 
+ - **Docs:** Contribute to the [Docs](docs/) or file issues for any docs bugs or requests [here](https://github.com/solo-io/service-mesh-hub/issues). 
+ - **Talks and Blogs:** If you are interested in writing or speaking about Service Mesh Hub and would like access to content, images or help, [DM us here](https://solo-io.slack.com/archives/DHQ9J939V). Share your demos, tutorials and content back to the community.
+
+### Community Meetings 
+Calls will be held every other Wednesday at **10am Pacific Time** and are open to the general public. These meetings will be recorded and posted publicly to YouTube. 
+ - [Zoom meeting link](https://solo.zoom.us/j/98337720715) - open to the public and recorded
+ - [Meeting Calendar](https://calendar.google.com/calendar/embed?src=solo.io_c144salt3ffnlfto3p1qnkbmdo%40group.calendar.google.com&ctz=America%2FLos_Angeles)
```
Comment (bettyjunod) [inline README.md:59]: changed the calendar link

**Pull request merged**